### PR TITLE
Add parenthesis to assert to avoid using `\`

### PR DESCRIPTION
**Pull request by JanuszL**
- follow-up for https://github.com/NVIDIA/DALI/pull/4120
- removes parenthesis around the error message in the assert

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- follow-up for https://github.com/NVIDIA/DALI/pull/4120
- removes parenthesis around the error message in the assert

### Affected modules and functionalities:
- docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py

### Key points relevant for the review:
- NA

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py [hunk 1]
```diff
@@ -325,9 +325,9 @@ def _get_caption_annotation(image_id):
 
 def main(_):
     assert FLAGS.image_dir, "`image_dir` missing."
-    assert FLAGS.image_info_file or \
-           FLAGS.object_annotations_file or \
-           FLAGS.caption_annotations_file, ("All annotation files are " "missing.")
+    assert (FLAGS.image_info_file or
+            FLAGS.object_annotations_file or
+            FLAGS.caption_annotations_file, "All annotation files are " "missing.")
```
Comment (klecki) [inline docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py:330]:
```suggestion
            FLAGS.caption_annotations_file, "All annotation files are missing.")
```

Comment (mzient) [inline docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py:330]:
```suggestion
            FLAGS.caption_annotations_file, "All annotation files are missing.")
```

**Review by klecki — APPROVED** [review]
Please fix the message, otherwise looks ok

File: docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py (hunk 1 above)
Comment (JanuszL) [reply docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py]: Done

Comment (JanuszL) [reply docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py]: Done

**Review by mzient — APPROVED** [review]

**Pull request merged**